### PR TITLE
Restructure msgq

### PR DESCRIPTION
**Pull request by haraschax**

### CODE REVIEW

File: SConstruct [hunk 1]
```diff
@@ -359,8 +359,12 @@ gpucommon = [_gpucommon]
 Export('common', 'gpucommon')
 
 # Build cereal and messaging
```
Comment (adeebshihadeh) [inline SConstruct:361]:
```suggestion
# Build messaging (cereal + msgq + socketmaster + their dependencies)
```

File: system/webrtc/tests/test_stream_session.py [hunk 2]
```diff
@@ -68,6 +68,8 @@ def test_incoming_proxy(self, mocker):
 
       mocked_pubmaster.reset_mock()
 
+  '''
+  FIXME, hangs for some reason
```
Comment (adeebshihadeh) [inline system/webrtc/tests/test_stream_session.py:72]: usually a skip is added instead. then the test still shows up in collection and you don't have to comment out all the imports

example: https://github.com/commaai/openpilot/blob/541b5cbf1387df07e76da55d3e332170d9e756f0/system/updated/casync/tests/test_casync.py#L16

File: .gitmodules [hunk 3]
```diff
@@ -5,7 +5,7 @@
   path = opendbc
   url = ../../commaai/opendbc.git
 [submodule "msgq"]
-  path = msgq
+	path = msgq_repo
```
Comment (adeebshihadeh) [inline .gitmodules:8]: tab?

**Comment by adeebshihadeh** [comment]
I removed Dockerfie.sim here: https://github.com/commaai/openpilot/commit/541b5cbf1387df07e76da55d3e332170d9e756f0

File: .gitmodules (hunk 3 above)
Comment (haraschax) [reply .gitmodules:8]: This happens when you do `git mv` btw that's why I keep hitting it

**Pull request merged**